capstone/src/main/java/com/google/univiz/UniversityData.java

```diff
@@ -0,0 +1,110 @@
+package com.google.univiz;
+
+public final class UniversityData {
+
+  public final class UniversityBuilder {
+    private String uniName;
+    private String uniCity;
+    private float latitude;
+    private float longitude;
+    private int isMainCampus;
+    private int urbanizationDegree;
+    private float admissionRate;
+    private float avgSAT;
+    private int numOfUndergrads;
+    private int avgCostOfAttendance;
+    private int numOfStudentsInDebt;
+    private int carnegieSizeDegree;
+    private float numOfUndergradMen;
+    private float numOfUndergradWomen;
+
+    public UniversityBuilder setName(String name) {
```

Comment:
can there be one function which is just one massive constructor so you can set all these fields at once; like a `public UniversityBuilder setAll(name, uniCity, ...)`